core/model/modx/processors/resource/update.class.php

```diff
@@ -452,6 +452,18 @@ public function checkForUnPublishOnSiteStart() {
      * @return boolean
      */
     public function checkDeletedStatus() {
+	    if (1 == $this->getProperty('deleted', null)) {
+	    	if ($this->modx->getOption('site_start') == $this->getProperty('id')) {
```

Comment:
I will take a look to it later this week when I have some time, I will fix them both.

I already use 4 spaces for the tab identation (Coda).